src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -140,296 +96,65 @@ public ResponseEntity<List<UUID>> getModificationGroups() {
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The network modification was updated")})
+    public ResponseEntity<Void> updateNetworkModification(
+            @Parameter(description = "Network modification ID") @PathVariable("networkModificationUuid") UUID networkModificationUuid,
+            @RequestBody ModificationInfos modificationInfos) {
+        // temporary switch, should be removed when all kind of modification will use the generic updateModification
+        // PS : the switch can't be in the service because of @Transactional that need to be called from outside the class
+        switch (modificationInfos.getType()) {
+            case LOAD_CREATION:
+            case LINE_SPLIT_WITH_VOLTAGE_LEVEL:
+                networkModificationService.updateModification(networkModificationUuid, modificationInfos);
+                break;
+            default:
+                networkModificationService.updateNetworkModification(networkModificationUuid, modificationInfos);
+        }
         return ResponseEntity.ok().build();
     }
 
-    @PostMapping(value = "/networks/{networkUuid}/lines-attach-to-split-lines", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
-    @Operation(summary = "attach lines to a split lines")
-    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The lines has been attached to split lines")})
-    public ResponseEntity<List<ModificationInfos>> createLinesAttachToSplitLinesCreation(@Parameter(description = "Network UUID") @PathVariable("networkUuid") UUID networkUuid,
-                                                                                         @Parameter(description = "Variant Id") @RequestParam(name = "variantId", required = false) String variantId,
-                                                                                         @RequestParam(value = "group", required = false) UUID groupUuid,
-                                                                                         @RequestParam(value = "reportUuid") UUID reportUuid,
-                                                                                         @RequestParam(value = "reporterId") String reporterId,
-                                                                                         @RequestBody LinesAttachToSplitLinesInfos linesAttachToSplitLinesInfos) {
-        return ResponseEntity.ok().body(networkModificationService.createLinesAttachToSplitLinesCreation(networkUuid, variantId, groupUuid, reportUuid, reporterId, linesAttachToSplitLinesInfos));
+    @GetMapping(value = "/network-modifications/{networkModificationUuids}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Get network modifications")
+    @ApiResponse(responseCode = "200", description = "The network modifications were returned")
+    public ResponseEntity<List<ModificationInfos>> getNetworkModifications(
```

Comment:
Rename also `getModifications` by `getNetworkModifications` for` /groups/{groupUuid}/modifications`

Reply:
So should I rename your new methods getModifications too ?